store/src/java/com/zimbra/cs/account/ZimbraAuthToken.java

```diff
@@ -376,8 +375,8 @@ public String getCrumb() throws AuthTokenException {
 
     @Override
     public boolean isRegistered() {
-        if (!isZimbraUser() || isZMGAppBootstrap()) {
-            return true;
+        if (!isZimbraUser()) {
+            return false;
```

Comment:
Why does it need to be changed to `false`?
The change on https://bugzilla.zimbra.com/show_bug.cgi?id=97110 just added ` || isZMGAppBootstrap()`. So can we just remove it?

Part of changes on bug 97110:
```
     public boolean isRegistered() {
-        if (!isZimbraUser()) {
+        if (!isZimbraUser() || isZMGAppBootstrap()) {
             return true;
         }
```

Reply:
This has been fixed in https://github.com/Zimbra/zm-mailbox/pull/1570/commits/fd8b6792e7359c54d2ae30e9ac51e3a70df1d196#diff-ec6294f6a1ccd7dfc9853dc54e1f74dc50cf2786f92e7f670449d17c6a2d9a23L379